src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -92,9 +92,9 @@
   final Logger LOGGER = LoggerFactory.getLogger(getClass());
   private static final String AGGREGATORS_MODULE_KEY = "AGGREGATORS";
 
-  private static final boolean failOnStartIfNoQuorum =
+  private final boolean failOnStartIfNoQuorum =
```

Comment:
for the purposes of the test, to test the coverage with the system property, if I have it final static, the test has no effect since it is declared as a Class Constant, the only way I know of to do it is using Reflection but in the initiative From Java17 we are reporting many problems with this practice. So we decided to make it an Instance constant by removing the static modifier.